book.toml

```
@@ -13,3 +13,42 @@ command = "mdbook-mermaid"
 
 [output.html]
 additional-js = ["doc/mermaid.min.js", "doc/mermaid-init.js"]
+
+[output.linkcheck]
+# Should we check links on the internet? Enabling this option adds a
+# non-negligible performance impact
+follow-web-links = false
```

Still open: did you try it?

```suggestion
follow-web-links = true
```

---

Yes, too many errors, I opened a follow up issue to enable this.